client/app/styles/_base.scss

```diff
@@ -241,7 +241,8 @@ a {
 
 .footer {
   @extend .overlay-wrapper;
-  right: 43px;
+  right: -380px;
+  bottom: -3px;
```

Comment:
bottom -3px not require here.

Reply:
The bottom function is resolve from the footer section.